packages/shared/invokeGuardedCallback.js

```diff
@@ -22,6 +22,8 @@ let invokeGuardedCallback = function<A, B, C, D, E, F, Context>(
 ) {
   this._hasCaughtError = false;
   this._caughtError = null;
+  this._hasSuppressedError = false;
```

Comment:
I did added `_hasSuppressedError` and `_suppressedError` to store the suppressed error and state, but can not figure out what to do with these fields.